_posts/2016-01-15-GitHubAnimator.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+name: GitHubAnimator
+title: 'GitHubAnimator'
+date: 2016-01-15 00:20:00 +00:00
+author: Pedro Santos
+image:
+    src: /assets/img/custom/blog/octokit.png
+tags:
+- GitHub 
+- animation
+- presentations
+- automation
+---
+
+### The Beginning
+Inspired by my friend [@SamirTalwar](https://twitter.com/SamirTalwar) and the initiative he set upon himself to [automate the world](https://noodlesandwich.com/]) I picked up on an idea I had many months ago. Automating the generation of presentations using GitHub commits.
+
+### What Problem am I Trying to Solve?
+Often I need to do a software related session and I have to present the code in small steps showing how I approached a problem. This usually involves a lot of copy and pasting code from a repository, a commit a time, into PowerPoint or KeyNote or some other presentation application or framework. This is a very manual process and error-prone. Also, presentation applications usually know nothing about code, so I also need to hand format the copied code.
+
+I've have had the idea of automating this process but kept delaying it for some reason or another. After reading Samir's posts I finally decided to get it done.
+
+### Implementation
+#### THE LANGUAGE
+I've been learning F# and I decided this would be a perfect opportunity to do something useful with my new F# skills. Also it looked like this problem was very suited to a functional approach.
```

Comment:
(maybe, to be more assertive): Also, this problem is very suited to a functional approach